entity-framework/core/logging-events-diagnostics/event-counters.md

```diff
@@ -70,14 +70,16 @@ Press p to pause, r to resume, q to quit.
 
 ## Counters and their meaning
 
-Counter name                          | Description
-------------------------------------- | ----
-Active DbContexts                     | The number of active, undisposed DbContext instances currently in your application. If this number grows continuously, you may have a leak because DbContext instances aren't being properly disposed. Note that if [context pooling](xref:core/performance/advanced-performance-topics#dbcontext-pooling) is enabled, this number includes pooled DbContext instances not currently in use.
-Execution Strategy Operation Failures | The number of times a database operation failed to execute. If a retrying execution strategy is enabled, this includes each individual failure within multiple attempts on the same operation. This can be used to detect transient issues with your infrastructure.
-Optimistic Concurrency Failures       | The number of times `SaveChanges` failed because of an optimistic concurrency error, because data in the data store was changed since your code loaded it. This corresponds to a <xref:Microsoft.EntityFrameworkCore.DbUpdateConcurrencyException> being thrown.
-Queries                               | The number of queries executed.
-Query Cache Hit Rate (%)              | The ratio of query cache hits to misses. The first time a given LINQ query is executed by EF Core (excluding parameters), it must be compiled in what is a relatively heavy process. In a normal application, all queries are reused, and the query cache hit rate should be stable at 100% after an initial warmup period. If this number is less than 100% over time, you may experience degraded perf due to repeated compilations, which could be a result of suboptimal dynamic query generation.
-SaveChanges                           | The number of times `SaveChanges` has been called. Note that `SaveChanges` saves multiple changes in a single batch, so this doesn't necessarily represent each individual update done on a single entity.
+The following counters are available and maintained in [`EntityFrameworkEventSource.cs`](https://github.com/dotnet/efcore/blob/main/src/EFCore/Infrastructure/EntityFrameworkEventSource.cs)
```

Comment:
Cannot add link to source here.

Reply:
Okay, ill remove it. Just figured I would keep it similar to this https://docs.microsoft.com/en-us/dotnet/core/diagnostics/available-counters